esx_service/utils/vmdk_utils.py

```diff
@@ -274,26 +277,25 @@ def get_vm_uuid_by_name(vm_name):
     """ Returns vm_uuid for given vm_name, or None """
     si = vmdk_ops.get_si()
     try:
-        vm = [d for d in si.content.rootFolder.childEntity[0].vmFolder.childEntity if d.config.name == vm_name]
-        return vm[0].config.uuid
+        vm = FindChild(GetVmFolder(), vm_name)
```

Comment:
Is GetVmFolder() defined in the ESX service, searched but not able to find the description for this. Which VM folder does this return data for?